encoding/codecv0/codecv0.go

```diff
@@ -141,6 +141,14 @@ func NewDAChunk(chunk *encoding.Chunk, totalL1MessagePoppedBefore uint64) (*DACh
 
 // Encode serializes the DAChunk into a slice of bytes.
 func (c *DAChunk) Encode() ([]byte, error) {
+	if len(c.Blocks) == 0 {
```

Comment:
If checked in `NewDAChunk`, why do another check here?

Reply:
I think it's not worse to also add a check here. since DAChunk may be initialized by other means in unit tests.